ej2-vue/daterangepicker/getting-started.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with Vue Daterangepicker component | Syncfusion
-description:  Checkout and learn about Getting started with Vue Daterangepicker component of Syncfusion Essential JS 2 and more details.
+title: Getting Started with Vue Date Range Picker Component | Syncfusion
+description: Learn here all about getting started with Syncfusion Essential Vue Date Range Picker component, it's elements and more.
```

Comment:
Revert this and remove unwanted space only.

Reply:
Reverted